{{cookiecutter.pipeline_slug}}/nextflow.config

```diff
@@ -64,11 +65,11 @@ trace {
 }
 dag {
   enabled = true
-  file = "${params.outdir}/pipeline_info/{{ cookiecutter.pipeline_name.replace(' ', '-') }}_dag.html"
+  file = "${params.outdir}/pipeline_info/{{ cookiecutter.pipeline_name.replace(' ', '-') }}_dag.svg"
 }
 
 manifest {
-  homePage = '{{ cookiecutter.pipeline_url }}'
+  homePage = 'https://github.com/nf-core/"{{ cookiecutter.pipeline_name }}"'
```

Comment:
I don't think you want this change. The cookiecutter template already suggests GitHub, so `{{ cookiecutter.pipeline_url }}` should already be good.

Reply:
Yup, but now the test for that manifest.homePage fails, but maybe thats a problem of the linting application and not of the cookiecutter tool.